.golangci.yml

```diff
@@ -76,7 +71,6 @@ linters:
     - errcheck
     - exportloopref
     - funlen
-    - gci
```

Comment:
Is there a particular reason why you do not want to run gci as part of the standard tests? Especially considering that it caught a case not covered by goimports. With these settings it should do the same thing as goimports.

Reply:
The CI runs the previous and the "new/local" version of golangci-lint, so it's not possible to apply a linter when there is introduced or "updated".
Also, the previous/current version gci, inside golangci-lint, doesn't work well with the project. Maybe in the future we will enable it.